docs/operators/jsonParser.md

```diff
@@ -0,0 +1,31 @@
+Json Parser
+=============
+
+## Operator Objective
+Purpose of this operator is to parse JSON records and construct a Plain Old Java Object ("POJO") out of it. The operator also emits each record as JSONObject. User can also provide a schema describing JSON data to validate incoming JSON records. Valid records will be emitted as POJO / JSONOnject while invalid ones are emitted on error port with error message.
+
+Json Parser does not hold any state and is **idempotent**, **fault-tolerance** & **statically/dynamically partitionable**.
+
+## Properties, Attributes and Ports
+### <a name="props"></a>Properties of Json Parser
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *jsonSchema* | [Schema](http://json-schema.org/)  describing JSON data | String | NO | N/A |
+
+### Platform Attributes that influences operator behavior
+| **Attribute** | **Description** | **Type** | **Mandatory** |
+| -------- | ----------- | ---- | ------------------ |
+| *out.TUPLE_CLASS* | TUPLE_CLASS attribute on output port which tells operator the class of POJO which need to be emitted | Class| Yes |
+
```

Comment:
Describe what the effect is of not providing jsonSchema. Also, what, if any, is the expected relationship between the jsonSchema and the TUPLE_CLASS ? More specifically, if the tuple has a property that is absent is the class, will it be ignored or throw an error ? Convesely, if the class has a field that is not present in the tuple what happens ?